### PR TITLE
Documented the build in IHitshap types

### PR DESCRIPTION
**Pull request by Mailaender**
Fills the gap at https://docs.openra.net/en/latest/playtest/traits/#hitshape

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/HitShape.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public class HitShapeInfo : ConditionalTraitInfo, Requires<BodyOrientationInfo>
 		public readonly BitSet<ArmorType> ArmorTypes = default(BitSet<ArmorType>);
 
 		[FieldLoader.LoadUsing("LoadShape")]
+		[Desc("Comes with support for `Circle`, `Polygon` and `Rectangle`. Defaults to `Circle` when left empty.")]
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/HitShape.cs]: Mods can add their own, so this list isn't necessary complete.

Comment (Mailaender) [reply OpenRA.Mods.Common/Traits/HitShape.cs]: I know.

**Comment by tovl** [comment]
You left out 'Capsule'.

**Comment by Mailaender** [comment]
Updated.

**Comment by deleted-user-1** [comment]
Commit message suggestion: `Add documentation for default IHitShape types.`

**Review by tovl — APPROVED** [review]

**Review by abcdefg30 — APPROVED** [review]
Lgtm.

**Pull request merged**